data_prep/generate_climos.py

```diff
@@ -5,37 +5,22 @@
 
 from argparse import ArgumentParser
 from datetime import datetime
-from tempfile import NamedTemporaryFile
 
 from cdo import Cdo
-from netCDF4 import Dataset, num2date, date2num
+from netCDF4 import date2num
 from dateutil.relativedelta import relativedelta
 
-from Cmip5File import Cmip5File, ClimdexFile
+from util import s2d, ss2d, d2ss, d2s
```

Comment:
These are now in `nchelpers`, right? Should they be exposed/exported from `nchelpers` or is the fact that we need them here indicative that we haven't accomplished proper encapsulation in `nchelpers`? Or... *do* we even need them here?